model/clusters_mgmt/v1/add_on_type.model

```diff
@@ -52,10 +52,12 @@ class AddOn {
 	// The name of the operator installed by this add-on.
 	OperatorName String
 
-	// Indicates if this add-on has external resources associated with it 
+	// Indicates if this add-on has external resources associated with it
 	HasExternalResources Boolean
 
 	// List of parameters for this add-on.
 	link Parameters []AddOnParameter
 
+	// List of requirements for this add-on.
+	link Requirements []AddOnRequirement
```

Comment:
The `link` is intended for when the actual collection is in another enpoint, but there is no such endpoint, or at least I can't find it.  The same applies to the existing `Parameters` attribute above.